Tests/StringTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  StringTests.swift
+//  Tests
+//
+//  Created by Igor Ranieri on 23.05.19.
+//
+
+import XCTest
+@testable import SweetSwift
+
+class StringTests: XCTestCase {
+    let prefix = "prefix:"
+
+    func testPrepending() {
+        var mutableString = "a string that I can change"
+
+        XCTAssertFalse(mutableString.hasPrefix(prefix))
+        mutableString.prepend(prefix)
+        XCTAssertTrue(mutableString.hasPrefix(prefix))
```

Comment:
In both of these tests, it's probably also worth testing the full string is what you expect it to be just to validate that you didn't do something silly like dropping the bit that you were trying to append